guides/release/components/conditional-content.md

```diff
@@ -54,52 +54,88 @@ We can do this with an `if`:
 
-In our case, if the `@localTime` property exists with truthy value then, the markup (`<span class="local-time">their local time is {{@localTime}}</span>`) will be rendered.
+Like any programming language, Ember also allows you to write `if-else` and `if-else if` statements in a template.
 
-If a value passed to `{{#if}}` evaluates to falsy, the `{{else}}` block
-of that invocation is rendered:
+```handlebars {data-filename="app/components/my-component.hbs"}
+{{#if condition}}
+  {{!-- some content --}}
+{{else}}
+  {{!-- some other content --}}
+{{/if}}
 
-```handlebars
-<h4 class="username">
-  {{@name}}
-  {{#if @localTime}}
-    <span class="local-time">their local time is {{@localTime}}</span>
-  {{else}}
-    Unable to fetch local time!
-  {{/if}}
-</h4>
+{{#if condition1}}
+  ...
+{{else if condition2}}
+  ...
+{{else if condition3}}
+  ...
+{{else}}
+  ...
+{{/if}}
 ```
 
+
 ## Inline `if`
 
-Sometimes, you need to place conditional content inside an argument or
-attribute. You can do this with _inline_ `if`. For instance, consider these two
-avatar components, for received messages and for sent messages respectively:
+### Motivation
+
+Sometimes, you will want to conditionally set an argument or attribute.
+
+For instance, consider two components that display a user's avatar. One is for a recipient and the other for a sender.
 
 ```handlebars {data-filename="app/components/received-message/avatar.hbs"}
 <aside>
-  <div class="avatar is-active" title="Tomster's avatar">T</div>
+  <div
+    class="avatar is-active"
+    title="Tomster's avatar"
+  >
+    T
+  </div>
 </aside>
 ```
 
 ```handlebars {data-filename="app/components/sent-message/avatar.hbs"}
 <aside class="current-user">
-  <div class="avatar" title="Zoey's avatar">Z</div>
+  <div
+    class="avatar"
+    title="Zoey's avatar"
+  >
+    Z
+  </div>
 </aside>
 ```
 
-The first component needs to have an `is-active` class applied to the avatar to
-show the active icon. That icon could also change over time, so it probably
-doesn't make sense to try to use `...attributes` to apply it. It's a part of the
-_public API_ of the component, so an argument makes sense.
+Again, the two components look similar. The first component has an `is-active` class, while the second a `current-user` class. How should we unify the components into one?
 
-So, we want to add the `is-active` class if an argument, like say `@isActive`,
-is passed in and is truthy.
+The `is-active` class is responsible for showing the active icon. _How_ that icon is rendered may change over time, so we won't use `...attributes` to apply the `is-active` class. Instead, we'll pass the argument `@isActive` to dictate _what_ to do (e.g. render the icon).
+
+As for the `current-user` class, it may have been just one of a few classes that can be applied to the `<aside>` element. Let's use `...attributes` to apply the `current-user` class.
```

Comment:
```suggestion
As for the `current-user` class, it may have been just one of a few classes that can be applied to the `<aside>` element.
Let's use `...attributes` to apply the `current-user` class.
```